ui/components/resolvers/HookFormGroup.js

```diff
@@ -1,12 +1,19 @@
 import React, { Component } from "react";
+import axios from "axios";
 import {
     Col,
-    FormControl
+    FormControl,
+    InputGroup,
+    Button,
+    Icon
 } from "patternfly-react";
+import { Validate, Validators } from "../../helper/Validators";
 
-import { hookControlLabel } from "./HookFormGroup.css";
+import { verifyButton, verifyButtonLabel } from "./HookFormGroup.css";
```

Comment:
Do we normally only import the styles we want? If the linked styles file is only for this js file anyway would it not make sense to import the whole file?

Reply:
Ah, I see why you do it this way now. On the mobile client service we used to not name the css since each css file directly corresponded to a javascript file it wouldn't be used anywhere else.

So:

`import './SomeCSSFile.css';`

and then you can just use any class from that file. I think in larger files a blanket import makes sense, not against naming the import though.